### PR TITLE
OpenMetadata install script

### PR DESCRIPTION
**Pull request by cdoron**

### CODE REVIEW

File: third_party/openmetadata/prepare_OM_for_fybrik.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 IBM Corp.
```
Comment (roytman) [inline third_party/openmetadata/prepare_OM_for_fybrik.go:1]: why do you need this start connector here ?

Comment (cdoron) [reply third_party/openmetadata/prepare_OM_for_fybrik.go:1]: The NewOpenMetadataAPIService() method runs a prepareOpenMetadataForFybrik() method which creates the custom properties that we need.

File: third_party/openmetadata/prepare_OM_for_fybrik.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+package main
+
+import (
+	logging "fybrik.io/fybrik/pkg/logging"
+	openapi_connector_core "fybrik.io/openmetadata-connector/pkg/openmetadata-connector-core"
+)
+
+func main() {
+	conf := map[interface{}]interface{}{
+		"openmetadata_endpoint": "http://localhost:8585/api",
```
Comment (roytman) [inline third_party/openmetadata/prepare_OM_for_fybrik.go:13]: can it be an argument or env parameter

File: third_party/openmetadata/prepare_OM_for_fybrik.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+	}
+	logger := logging.LogInit(logging.CONNECTOR, "OpenMetadata Prepare for Fybrik")
+	openapi_connector_core.NewOpenMetadataAPIService(conf, &logger)
```
Comment (roytman) [inline third_party/openmetadata/prepare_OM_for_fybrik.go:16]: can we separate the connection and tag creation from NewOpenMetadataAPI Service creation, and get the custom properties and tags from external files

File: third_party/openmetadata/prepare_OM_for_fybrik.go (hunk 1 above)
Comment (cdoron) [reply third_party/openmetadata/prepare_OM_for_fybrik.go]: I removed the go code.
The preparation is now done using the same executable as that of the openmetadata-connector (with different parameters).

File: third_party/openmetadata/prepare_OM_for_fybrik.go (hunk 2 above)
Comment (cdoron) [reply third_party/openmetadata/prepare_OM_for_fybrik.go]: The preparation code gets the endpoint, username, and password from an environment variable.

File: third_party/openmetadata/prepare_OM_for_fybrik.go (hunk 3 above)
Comment (cdoron) [reply third_party/openmetadata/prepare_OM_for_fybrik.go]: done.

File: third_party/openmetadata/install_OM.sh [hunk 4]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+OM_VERSION=0.12.1
+export OPENMETADATA_AIRFLOW_VERSION=0.12.1
+export OPENMETADATA_HELM_CHART_VERSION=0.0.39
+
+export export FYBRIK_BRANCH="${FYBRIK_BRANCH:-master}"
+export export FYBRIK_NAMESPACE="${FYBRIK_NAMESPACE:-fybrik}"
```
Comment (roytman) [inline third_party/openmetadata/install_OM.sh:8]: I think its name is organization (guthub organization) and not a namespace

File: third_party/openmetadata/install_OM.sh [hunk 5]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+OM_VERSION=0.12.1
+export OPENMETADATA_AIRFLOW_VERSION=0.12.1
+export OPENMETADATA_HELM_CHART_VERSION=0.0.39
+
+export FYBRIK_BRANCH="${FYBRIK_BRANCH:-master}"
+export FYBRIK_GITHUB_ORGANIZATION="${FYBRIK_NAMESPACE:-fybrik}"
```
Comment (roytman) [inline third_party/openmetadata/install_OM.sh:8]: ${FYBRIK_NAMESPACE:-fybrik} -> ${FYBRIK_GITHUB_ORGANIZATION:-fybrik}

**Review by roytman — APPROVED** [review]

**Pull request merged**